### PR TITLE
Added an ErrorLogWriter for Laminas\Log\Writer\

### PR DESCRIPTION
**Pull request by robert-van-de-coolwijk**

### CODE REVIEW

File: src/Logger/ErrorLogWriter.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+         * @param  null|string $logSeparator Log separator string
+         * @param  null|int $filePermissions Permissions value, only applicable if a filename is given;
+         *     when $streamOrUrl is an array of options, use the 'chmod' key to specify this.
+         * @return Stream
+         * @throws Exception\InvalidArgumentException
+         * @throws Exception\RuntimeException
+         */
+        public function __construct()
+        {
+            if ($this->formatter === null) {
```
Comment (chris-kruining) [inline src/Logger/ErrorLogWriter.php]: please conform to code-style (all brackets on a new line)

File: src/Logger/ErrorLogWriter.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        /**
+         * Set log separator string
+         *
+         * @param  string $logSeparator
+         * @return Stream
+         */
+        public function setLogSeparator($logSeparator)
+        {
+            $this->logSeparator = (string) $logSeparator;
+            return $this;
```
Comment (chris-kruining) [inline src/Logger/ErrorLogWriter.php:63]: code-style: newline around keywords

File: src/Logger/ErrorLogWriter.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        /**
+         * Write a message to the log.
+         *
+         * @param array $event event data
+         * @return void
+         * @throws Exception\RuntimeException
+         */
+        protected function doWrite(array $event)
+        {
+            $line = $this->formatter->format($event) . $this->logSeparator;
```
Comment (chris-kruining) [inline src/Logger/ErrorLogWriter.php]: can be placed inside the function call

File: src/Logger/ErrorLogWriter.php (hunk 1 above)
Comment (robert-van-de-coolwijk) [reply src/Logger/ErrorLogWriter.php]: Done

File: src/Logger/ErrorLogWriter.php (hunk 3 above)
Comment (robert-van-de-coolwijk) [reply src/Logger/ErrorLogWriter.php]: Done

File: src/Logger/ErrorLogWriter.php (hunk 2 above)
Comment (robert-van-de-coolwijk) [reply src/Logger/ErrorLogWriter.php:63]: Done

**Review by chris-kruining — APPROVED** [review]

**Pull request merged**